tests/tests.py

```diff
@@ -32,6 +36,7 @@ def setUp(self):
         # User ID 33333333-3333-3333-3333-333333333333 receives alternative 1
 
     def test_user_is_assigned_user_id(self):
+        print('test_user_is_assigned_user_id')
```

Comment:
stray print?

Reply:
Yes, it will be removed with the next update.